server/lib/availabilitySync.ts

```diff
@@ -312,67 +316,49 @@ class AvailabilitySync {
       try {
         // Check if both exist or if a single non-4k or 4k exists
         // If both do not exist we will return false
-
-        let meta: RadarrMovie | undefined;
-
         if (!server.is4k && media.externalServiceId) {
-          meta = await api.getMovie({ id: media.externalServiceId });
+          radarr = await api.getMovie({ id: media.externalServiceId });
         }
 
         if (server.is4k && media.externalServiceId4k) {
-          meta = await api.getMovie({ id: media.externalServiceId4k });
-        }
-
-        if (!server.is4k && (!meta || !meta.hasFile)) {
-          existsInRadarr = false;
-        }
-
-        if (server.is4k && (!meta || !meta.hasFile)) {
-          existsInRadarr4k = false;
+          radarr4k = await api.getMovie({ id: media.externalServiceId4k });
```

Comment:
For the radarr and sonarr functions, the returned data has been moved to outside of the for loop for better checking if the movie/show exists. This also prevents a bug that was causing single server users to show that they had a 4k server or vise versa.